src/audio/drc/drc_math_generic.c

```diff
@@ -229,11 +234,11 @@ inline int32_t drc_inv_fixed(int32_t x, int precision_x, int precision_y)
 	if (sqrt2_extracted)
 		inv = q_mult(inv, SQRT2, qc, 30, qc);
 
-	e += qc;
-	if (e > precision_y)
-		return Q_SHIFT_RND(inv, e, precision_y);
-	if (e < precision_y)
-		return Q_SHIFT_LEFT(inv, e, precision_y);
+	precision_inv = e + qc;
+	if (precision_inv > precision_y)
+		return Q_SHIFT_RND(inv, precision_inv, precision_y);
+	if (precision_inv < precision_y)
+		return Q_SHIFT_LEFT(inv, precision_inv, precision_y);
 	return inv;
 #undef qc
```

Comment:
please change qc to QC

Reply:
this is outside the scope of the review, would need another commit/PR

Reply:
Ok, then I think current modification is good enough to address inverse bug